TreeTracker/Resources/Localizable.strings

```diff
@@ -99,3 +100,19 @@
 "Settings.PhotoQuality.Option.Medium.Info" = "Medium resolution, medium data usage.";
 "Settings.PhotoQuality.Option.High.Title" = "High";
 "Settings.PhotoQuality.Option.High.Info" = "High resolution, high data usage.";
+
+//ChatList Screen
+"ChatList.NoMessagesLabel.Text" = "No messages, Yet.";
```

Comment:
I think we can drop the "Yet". Simply "No messages". Otherwise a user may be expecting something that may never come.

Reply:
Yep, I agree with you. I'll make the change.